games-simulation/simutrans/simutrans-120.2.1.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="A transport simulation game"
+DESCRIPTION="Simutrans is an open-source transportation simulation game \
+available for Windows, Mac, Linux and BeOS. Your goal is to establish a \
+successful transport company and build the necessary infrastructure to \
+transport passengers, mail and goods between cities, factories and \
+touristic attractions by land, air and water."
+HOMEPAGE="http://www.simutrans.com"
+SvnRevision="8121"
+SOURCE_URI="https://downloads.sourceforge.net/project/simutrans/simutrans/120-2-1/simutrans-src-120-2-1.zip"
+CHECKSUM_SHA256="7dfbd1c44f47ddd8caedb3da77dac8262652f96e1c9ffd890b67757e08ddc942"
+SOURCE_FILENAME="simutrans-$portVersion.zip"
+SOURCE_DIR=""
+COPYRIGHT="1997-2004 Hj. Malthaner
+	2005-2017 The Simutrans Team"
+LICENSE="Artistic"
+REVISION="1"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+PROVIDES="
+	simutrans$secondaryArchSuffix = $portVersion
+	"
```

Comment:
Okay, I had a look at a few other recipes that use "app:something" but I'm not sure if I should add "app:simutrans" (the file name of the actual binary is lower case) or "app:Simutrans" (as in the addAppDeskbarSymlink entry). I've also seen for "something$secondaryArchSuffix" packages entries with ("app:something$secondaryArchSuffix") and without ("app:something") $secondaryArchSuffix.